requirements.in

```diff
@@ -1,4 +1,5 @@
 ansible>=6,<7
+contextlib2>=21.6.0
```

Comment:
Isn't the [contextlib from standard library](https://docs.python.org/3/library/contextlib.html) enough for this?
It has the same contextmanager decorator you are using :thinking:

Reply:
yep, it is. 👍  I'll revert the requirement files.